packages/x-components/src/x-modules/search-box/components/search-input.vue

```diff
@@ -84,11 +84,12 @@
     public query!: string;
 
     /**
-     * Focus search input when the user navigates to the search input.
+     * Focus search input when the user navigates to the search input or
+     * when the user clicks on the clear search button.
      *
      * @internal
      */
-    @XOn('UserReachedEmpathizeTop')
+    @XOn(['UserReachedEmpathizeTop', 'UserPressedClearSearchBoxButton'])
```

Comment:
I know that it's out of the scope of this PR, but would you mind to to the same in the line 105?